simulator/infra/instrcache/t/unit_test.cpp

```diff
@@ -48,7 +48,7 @@ TEST_CASE( "check_method_size: Check_Method_Size")
     }
     CHECK( SIZE == instr_cache.size());
 
-    instr_cache.erase( PC - 1);
+    //instr_cache.erase( PC - 1);
```

Comment:
What is the purpose of these changes?

Reply:
Oh. I did that, because I had problems with build of project. Didn't want to commit it.